e2e/alphabet-card-detail.test.ts

```diff
@@ -41,4 +41,9 @@ describe('Detail', () => {
         await element(by.id('Next')).multiTap(26);
         await expect(element(by.id('apple'))).toBeVisible();
     });
+
+    it('should display the word audio play button', async () => {
```

Comment:
Can you also click the button and make sure that it doesn't display an error \ blow up?

Reply:
You haven't done this yet.

Reply:
It's clicking the play button that gives you confidence it won't blow up when your users click it.